lib/cfg-block-generator.c

```diff
@@ -39,6 +39,8 @@ _report_generator_args(gpointer key, gpointer value, gpointer user_data)
 {
   GString *result = (GString *) user_data;
   g_string_append_printf(result, "## %s=", (gchar *) key);
+  if (!value)
+    return;
```

Comment:
don't we need the trailing \n?

Reply:
I opted for fixing this at a different layer in the end (I think this is the right patch).